topology-bench.yaml

```diff
@@ -0,0 +1,19 @@
+nodes:
```

Comment:
Can you put config files (I guess I saw more such files earlier) into some dedicated directory (probably somewhere in `scripts`, but maybe somewhere else)?